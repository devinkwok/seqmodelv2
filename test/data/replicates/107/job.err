non-relevant line
non-relevant line

[2021-06-12 16:24:04,516] [CRITICAL] - seqmodel/run.py terminated with ERROR.
Traceback (most recent call last):
  File "seqmodel/run.py", line 176, in <module>
    Runner().run()
  File "seqmodel/run.py", line 172, in run
    raise e
  File "seqmodel/run.py", line 166, in run
    raise ValueError('TEST exception')
ValueError: TEST exception