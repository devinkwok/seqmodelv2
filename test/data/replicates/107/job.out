non-relevant line
non-relevant line

[2021-06-12 16:24:04,514] [INFO] - seqmodel/run.py started.
[2021-06-12 16:24:04,516] [CRITICAL] - seqmodel/run.py terminated with ERROR.
